dnsapi/dns_dnsever.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env sh
+
+#Here is a sample custom api script.
+#This file name is "dns_myapi.sh"
+#So, here must be a method   dns_myapi_add()
+#Which will be called by acme.sh to add the txt record to your api system.
+#returns 0 means success, otherwise error.
+#
+#Author: Neilpang
+#Report Bugs here: https://github.com/Neilpang/acme.sh
+#
+########  Public functions #####################
+
+# Please Read this guide first: https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide
+
+#Usage: dns_myapi_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_dnsever_add() {
+  fulldomain=$1
+  txtvalue=$2
+  _info "Using dnsever add"
+  _debug fulldomain "$fulldomain"
+  _debug txtvalue "$txtvalue"
+
```

Comment:
https://github.com/acmesh-official/acme.sh/blob/master/dnsapi/dns_cf.sh#L21

please use `_readaccountconf_mutable `  and `_saveaccountconf_mutable `

Reply:
fixed.